devicetypes/sky-nie/evalogik-door-window-sensor.src/evalogik-door-window-sensor.groovy

```diff
@@ -0,0 +1,616 @@
+/**
+ *     Evalogik Door/Window Sensor v1.0.1
+ *
+ *  	Models: MSE30Z/ZWS713
+ *
+ *  Author:
+ *   winnie (sky-nie)
+ *
+ *	Documentation:
+ *
+ *  Changelog:
+ *
+ *    1.0.1 (07/13/2021)
+ *      - Syntax format compliance adjustment
+ *      - delete dummy code
+ *
+ *    1.0.0 (04/26/2021)
+ *      - Initial Release
+ *
+ * Reference：
+ *   https://community.smartthings.com/t/release-aeotec-trisensor/140556?u=krlaframboise
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition(name: "Evalogik Door/Window Sensor",namespace: "sky-nie",author: "winnie",ocfDeviceType: "x.com.st.d.sensor.contact",runLocally: true,executeCommandsLocally: false,genericHandler: "Z-Wave") {
```

Comment:
Nit: add space after commas

Reply:
Does this DTH work in a full scope with runLocally flag set to true ?

Reply:
I removed runLocally  as IDE required. It seems to be no different from before.

Reply:
I removed runLocally  as IDE required. It seems to be no different from before.